From 0acf025d455a2180843135690930933a2266cf52 Mon Sep 17 00:00:00 2001
From: "Gary D. Gregory" <ggregory@apache.org>
Date: Tue, 22 May 2012 22:29:36 +0000
Subject: [PATCH] [VFS-417][RAM][Local] Add and implement new API:
 RandomAccessContent.setLength(long). [VFS-406][RAM] resize throws ArrayOOBE
 when shrinking in size.

git-svn-id: https://svn.apache.org/repos/asf/commons/proper/vfs/trunk@1341666 13f79535-47bb-0310-9956-ffa450edef68
---
 src/changes/changes.xml | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/src/changes/changes.xml b/src/changes/changes.xml
index f3c490aafe..0066d1246e 100644
--- a/src/changes/changes.xml
+++ b/src/changes/changes.xml
@@ -23,6 +23,12 @@
 
   <body>
     <release version="2.1" date="TBD" description="">
+      <action issue="VFS-417" dev="ggregory" type="add">
+        [RAM][Local] Add and implement new API: RandomAccessContent.setLength(long).
+      </action>
+      <action issue="VFS-406" dev="ggregory" type="fix" due-to="mp1">
+        [RAM] resize throws ArrayOOBE when shrinking in size.
+      </action>
       <action issue="VFS-321" dev="ggregory" type="update" due-to="sebb@apache.org">
         AbstractFileObject sometimes uses getFileSystem() and sometimes references "fs" field directly.
       </action>
